doc/releases/squid.rst

```diff
@@ -14,11 +500,11 @@ We recommend that all users update to this release.
 Notable Changes
 ---------------
 
-* This hotfix release resolves an RGW data loss bug when CopyObject is used to copy an object onto itself. 
```

Comment:
This is the release note for 19.2.2 which _was_ a hotfix release.

@yuriw It would be better to avoid unrelated whitespace changes in the PR.

Reply:
@yuriw Did you mean to resolve this comment without pushing any changes?  I don't see a reason to touch an old (19.2.2) release note in a PR that adds a new (19.2.3) release note.